config/initializers/inflections.rb

```diff
@@ -13,7 +13,11 @@
   inflect.acronym 'MDOT'
   inflect.acronym 'PagerDuty'
   inflect.acronym 'PPIU'
+  inflect.acronym 'SAML'
+  inflect.acronym 'SFTP'
+  inflect.acronym 'SM'
   inflect.acronym 'SSOe'
+  inflect.acronym 'SSO'
```

Comment:
I don't see any code updated as a result of this new inflection - what gives??

Reply:
Zeitwerk tries to link filenames to class names, so `sso_whatever` should have `SsoWhatever` as the classname unless you have the inflection.
However I can't find a real example of that here, and I tried my old zeitwerk PR without this inflection and it doesn't seem to cause issues.  So I'm going to take it out.